scripts/netdep_graph_json.py

```diff
@@ -46,6 +46,7 @@ def main():
   with open('clovis-trace.log') as f:
```

Comment:
oh I didn't see your other comment. The python files were already on master. Do you want me to remove them and put them on other branch? There is still use for them in the sense you can do `lighthouse --save-artifacts` and then use the artifacts through clovis to generate the dependency graph png picture. If you want I can send out a separate PR removing all python/clovis code.